server.js

```diff
@@ -199,11 +199,11 @@ app.get('/api/sms/', function(req, res) {
 app.get('/api/sms', function(req, res) {
   auth.check(req, 'can_access_gateway_api', null, function(err) {
     if (err) {
-      return serverUtils.error(err, req, res);
+      return serverUtils.error(err, req, res, true);
```

Comment:
Why do we want to `showPrompt` for these URLs?  AFAIK this is not required by gateway.

Or does this just mean "don't redirect to login page"?  If the latter, the parameter name in `serverUtils.error()` would benefit from being renamed.